package.json

```diff
@@ -83,6 +83,7 @@
     "katex": "^0.12.0",
     "linkifyjs": "^2.1.9",
     "lodash": "^4.17.20",
+    "matrix-analytics-events": "https://github.com/matrix-org/matrix-analytics-events.git",
```

Comment:
preferably this ends up on npm or at least tied to a commit, to avoid cases where we're debugging different checkouts and versions.

Reply:
How do we depend on the js-sdk? I think it's kind of a similar situation..

Reply:
During development the `package.json` points to `github:matrix-org/matrix-js-sdk#develop`. However the release script will replace that value with a version deployed to npm during the release cycle.

This happens in `element-web/release.sh` or `matrix-react-sdk/release.sh` for example

Reply:
also, when it's in `develop` we have the checkout linked anyways so it doesn't really affect much. The widget-api though doesn't traditionally get linked in dev, so we use an npm release. For the linter stuff, we point at a commit for reasons I cannot recall.